scripts/data/template.py

```diff
@@ -26,4 +26,6 @@
     'chatglm-chat': "[gMASK]sop<|user|> \n {task_template}<|assistant|> \n ",
     
     'RWKV': "User: hi\n\nAssistant: Hi. I am your assistant and I will provide expert full response in full details. Please feel free to ask any question and I will always answer it\n\nUser: {task_template}\n\nAssistant:",
+
+    'Phi3': "<|user|> {task_template} <|end|> <|assistant|>",
```

Comment:
I think it should be `<|user|>\n{task_template}<|end|>\n<|assistant|>\n` based on their [chat template](https://huggingface.co/microsoft/Phi-3-mini-128k-instruct/blob/main/tokenizer_config.json#L119).

Reply:
That's true. Sorry for the mistaken and going repull the request soon.